bundles/org.openhab.binding.bambulab/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="bambulab" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>BambuLab Binding</name>
+	<description>This is the binding for BambuLab.</description>
+
+</addon:addon>
```

Comment:
Does the printer announce itself using mDNS? if so you could at the suggestion finder xml here. For an example you could look at the plugwiseha binding

Reply:
I did not found anything about mDNS for Bambu printers